tasks/test.yaml

```diff
@@ -31,6 +32,15 @@ tasks:
       - task: create:k3d-standard-bundle
       - task: deploy:k3d-standard-bundle
       - task: validate-packages
+      - task: compliance:validate
+        with:
+          oscalfile: ./compliance/oscal-component.yaml
+          assessment_results: ./compliance/oscal-assessment-results.yaml
+      - task: compliance:evaluate
+        with:
+          assessment_results: ./compliance/oscal-assessment-results.yaml
+      - description: copy assessment file to log location
+        cmd: cp ./compliance/oscal-assessment-results.yaml /tmp/oscal-assessment-results.yaml
```

Comment:
I see this being added to the pipeline log artifacts, but wondering if there is a desire/intent to re-commit the result to the repo in a given PR as well?

Reply:
@CloudBeard had the same thought. 

I'm open that option - largely the guidance at a minimum should be to update your threshold anytime the compliance state is greater (better) than the current threshold. We're working on better ways to identify when that greater compliance state is available. 

That said - wanted to introduce the minimum with regards to workflows.

Reply:
Maybe to ask a slightly different question here - I see a committed `compliance/oscal-assessment-results.yaml`. If the one generated in CI differs from what is committed is there an expectation that we would update the committed one? Just want to understand the role of what is committed vs what is in CI.

Reply:
To capture intent: 

1. CI will generate a historical artifact (a new result in the assessment-results artifact) on each run with `lula validate`
2. if you look in the CI artifact that is uploaded - you will now see `two` results (the original threshold committed in this PR and the latest result established in the pipeline)
3. `lula evaluate` will compare the findings and pass the pipeline if the compliance is equal or greater - or fail the pipeline if compliance is lesser
4. The current guidance is that the CI artifact should only need to be re-committed to the repo when the latest `validate` produced a state of greater compliance (IE we need to set this as the new threshold)
5. Lula will automatically set the new threshold identifier in the CI artifact - so it just needs to be re-committed. 

There is no requirement currently that we store every single execution - maybe that changes in the future.

Reply:
Awesome, that helps my understanding a ton. I'll resolve this thread, I think the way you explained it makes sense that we wouldn't add an auto-commit here right now to simplify the changes.